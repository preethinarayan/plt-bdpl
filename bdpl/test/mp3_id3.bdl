struct MP3_FILE
{
	struct MP3_ID3
	{
		byte[3]	id;
		byte[2]	version;
		byte flags;
		int size;
	}mp3_id3;

	struct FRAME
	{
		byte[4] frame_id;
		byte[4] size;
		byte[2] flags;
    	struct TEXT_FRAME
	    {
		    byte encoding;
		    byte[size-1] string;
	    }text;
    }[10]textFrame;
}mp3_file;

file "x:/test/aa.mp3.mp3" inp_file;
print("finished reading input file ... printing output");
read(inp_file,mp3);
print(a);
