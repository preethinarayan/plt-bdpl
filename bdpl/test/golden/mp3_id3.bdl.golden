finished reading input file ... printing output